update getting started (use gwt-maven-archetype) (#354)
gwtproject/gwt-site: src/main/markdown/using-intellij.md

Using IntelliJ
===

GWT provides a set of tools that can simply be used with a
text editor, the command line, and a browser. However, you may also use GWT with your
favorite IDE. The Idea IntelliJ Ultimate Edition provides a GWT plugin, but the free Idea IntelliJ Community Edition 

> is all you need. IntelliJ provides an awesome Maven support. That is all, thatis  really necessary.

Reviewer: ```suggestion
is all you need. IntelliJ provides awesome Maven support, enabling the IDE to read the project classpath, and easily create run configurations for compiling and running a GWT project.
```

Reviewer: Probably should also add a note that Ultimate will add additional support for uibinder, jsni, .gwt.xml editing, but that these aren't required or necessary.

*   [Download IntelliJ](#intellij)
*   [Import a Web Application](#importing)
*   [Run locally in Super Dev Mode](#running)

## Download IntelliJ<a id="intellij"></a>

> If you do not already have IntelliJ, you may download it from the [IntgelliJ Website](https://www.jetbrains.com/idea/download/). 

Reviewer: ```suggestion
If you do not already have IntelliJ, you may download it from the [IntelliJ Website](https://www.jetbrains.com/idea/download/). 
```

## Import a Web Application<a id="importing"></a>

To import a Web Application - create with an archetype creator, select **File > New > Project from existing source** 
from the File menu. Select the root directory of the project you want to import and press `enter`.

A dialog opens:

<div class="screenshot"><a href="images/import-into-intellij.png"><img src="images/import-into-intellij.png" alt="Screenshot" width="42%"/></a></div>

> Press `crate`. The import process starts and after a few seconds, the project window appears:

Reviewer: ```suggestion
Press `Create`. The import process starts and after a few seconds, the project window appears:
```

<div class="screenshot"><a href="images/intellij-project.png"><img src="images/intellij-project.png" alt="Screenshot" width="42%"/></a></div>

## Run locally in Super Dev Mode<a id="running"></a>

> To improve the development experience, you can set up to run configurations, one for the code server and another for the server.

Reviewer: ```suggestion
To improve the development experience, you can set up two run configurations, one for the code server and another for the server.
```

### Code Server Run Configuration

> Press `edit configuration`, a popup appears. Now press `+` and select `Maven`.  

Reviewer: ```suggestion
Press `Edit Configuration`, a popup appears. Now press `+` and select `Maven`.  
```

> Enter `gwt:codeserver -pl mywebapp-client -am` in the field under run:

Reviewer: ```suggestion
Enter `gwt:codeserver -pl mywebapp-client -am` in the field under `Run`:
```

<div class="screenshot"><a href="images/intellij-run-code-server.png"><img src="images/intellij-run-code-server.png" alt="Screenshot" width="42%"/></a></div>

> Now, you have a run configuration, that starts the code server.

Reviewer: ```suggestion
Now you have a run configuration that starts the code server.
```

> Note: you have to use the module name instead of '*'.

Reviewer: I think you forgot to substitute the * above? It makes more sense below.

Author: added more text to clearify

### Server Run Configuration

> Press `edit configuration`, a popup appears. Now press `+` and select `Maven`.

Reviewer: ```suggestion
Press `Edit Configuration`, a popup appears. Now press `+` and select `Maven`.
```
